### DISCUSSION TITLE
Did anything change with Gitlab, Merge Requests, and Assignees?

### DISCUSSION THREAD

**Discussion by kjq** [renovatebot/renovate#23406] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

36.8.11

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

It seems like Renovate has changed how it handles assignees/reviewers when creating the GitLab request.  It used to only assign them if the MR failed but lately not only does it seem to assign them all the time but users are getting emails about MRs in projects they are not even associated to other than being a member of the group.

Is there a configuration that has changed on this I need to update?

### What you tried so far.

Looking for any configuration changes.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by RahulGautamSingh** [comment]
Is it possible for you to tell after which version update it stopped working?

**Reply by kjq** [reply, marked as answer]
I found half the problem so far ;)  Let me see what it looks like after it runs tonight.  We recently switched to a "group" for the GitLab email and anyone in that group was getting the emails.
